tests/bluetooth/bsim/nrf_auraconfig/tester/sysbuild/ipc_radio/prj.conf

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_MBOX=y
+CONFIG_IPC_SERVICE=y
+
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+
+CONFIG_BT_ISO_PERIPHERAL=y
+CONFIG_BT_ISO_SYNC_RECEIVER=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_PER_ADV_SYNC_TRANSFER_RECEIVER=y
+
+CONFIG_BT_CTLR_CONN_ISO_GROUPS=1
+CONFIG_BT_CTLR_CONN_ISO_STREAMS=2
+CONFIG_BT_CTLR_SYNC_ISO_STREAM_COUNT=2
+CONFIG_BT_CTLR_ADV_ISO_SET=1
+CONFIG_BT_CTLR_ADV_ISO_STREAM_COUNT=2
+
+# Support two links as a central, or one link as a peripheral
+CONFIG_BT_MAX_CONN=3
+CONFIG_BT_CTLR_SDC_PERIPHERAL_COUNT=1
+
+# Allow using more than default advertising event length
+CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=251
+
+# To present the audio at the right point in time, we need the controller and
+# audio clock to be synchronized
+CONFIG_MPSL_TRIGGER_IPC_TASK_ON_RTC_START=y
```

Comment:
Can these be removed?

Reply:
Will check

Reply:
It still worked with them removed, so I'll remove them from this PR